### PR TITLE
chore: add tf version variable

### PR DESCRIPTION
**Pull request by symeonmanis**

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -12,6 +12,9 @@ inputs:
     description: Terraform CLI args for init
   additional-apply-args:
     description: Terraform CLI args for apply
+  terraform-action:
```
Comment (daverichards-ellogroup) [inline action.yml]: The variable should be `terraform-version`, and the setup terraform action below would need to use the version too

**Comment by daverichards-ellogroup** [comment]
@symeonmanis you still need to add the `with` to the Setup Terraform job to include the version

**Comment by symeonmanis** [comment]
> @symeonmanis you still need to add the `with` to the Setup Terraform job to include the version

@daverichards-ellogroup added now

File: action.yml [hunk 2]
```diff
@@ -12,13 +12,18 @@ inputs:
...
+    default: latest
 
 runs:
   using: composite
 
   steps:
     - name: Setup Terraform
       uses: hashicorp/setup-terraform@v3
+      with:
+          terraform_version: "1.9.8"
```
Comment (daverichards-ellogroup) [inline action.yml]: You need to use the variable here, so the user of this action can configure it

**Review by daverichards-ellogroup — CHANGES_REQUESTED** [review]

**Review by daverichards-ellogroup — APPROVED** [review]

**Pull request merged**